prowler/providers/aws/services/wafv2/wafv2_service.py

```diff
@@ -35,6 +37,22 @@ def __list_web_acls__(self, regional_client):
                 f"{regional_client.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __get_logging_configuration__(self, regional_client):
+        logger.info("WAFv2 - Get Logging Configuration...")
+        try:
+            for acl in self.web_acls:
```

Comment:
Please include another `try/except` block here not to stop the loop if something is raised from within.